applications/daqiri_raw_ethernet_bench/cpp/main.cpp

```
@@ -29,35 +29,28 @@ class App : public holoscan::Application {
+      auto bench_tx = make_operator<ops::DaqiriRawEthernetBenchDefaultTxOp>(
+          "bench_tx",
+          from_config("bench_tx"),
+          make_condition<BooleanCondition>("is_alive", true));
+      add_operator(bench_tx);
+    }
 #else
-      HOLOSCAN_LOG_ERROR("DPDK manager/backend is disabled");
-      exit(1);
+    HOLOSCAN_LOG_ERROR("DPDK engine is disabled; rebuild with -DDAQIRI_ENGINE=dpdk");
+    exit(1);
 #endif
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If HOLOSCAN_OPERATOR_FORWARD_ARGS is a macro then please configure it.

(unknownMacro)